docs/manual/notifications/datadog.md

```diff
@@ -0,0 +1,46 @@
+# Datadog Notification Plugin (Enterprise)
+
+Enterprise customers have access to this plugin, that allows you to create an event in Datadog based on the execution status of a job. For example, you can configure it to create an event if the job failed or if the job succeeded.
+
+## Setup 
+
+To begin, you will need to add the API and APP to the project configuration setting, go to "Project Settings" and select "Edit Configuration." Next, select "Edit Configuarion File" and add the following:
+
+```bash
```

Comment:
Is this still needed or can it use the same framework.properties path statement?